### PR TITLE
WIP: FIPs Spike PR

### PR DESCRIPTION
**Pull request by eslutsky**

### CODE REVIEW

File: scripts/devenv-builder/configure-vm.sh [hunk 1]
```diff
@@ -101,25 +101,26 @@ fi
...
-    sudo tar -C "/usr/local/go${GO_VER}" -xzf "go${GO_VER}.linux-${GO_ARCH}.tar.gz" --strip-components 1
-    sudo rm -rfv /usr/local/bin/{go,gofmt}
-    sudo ln --symbolic /usr/local/go${GO_VER}/bin/{go,gofmt} /usr/local/bin/
-    rm -rfv "go${GO_VER}.linux-${GO_ARCH}.tar.gz"
-fi
+
+# update to golang from centos-9-stream
+    sudo tee "/etc/yum.repos.d/centos-9-stream.repo" >/dev/null <<EOF
+[appstream]
+name=CentOS Stream 9 - Appstream
```
Comment (ggiguash) [inline scripts/devenv-builder/configure-vm.sh:115]:
```suggestion
name=CentOS Stream 9 - AppStream
```

File: scripts/devenv-builder/configure-vm.sh [hunk 2]
```diff
@@ -101,25 +101,26 @@ fi
...
+baseurl=https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os
+gpgcheck=0
```
Comment (ggiguash) [inline scripts/devenv-builder/configure-vm.sh:117]: Does it work with gpgcheck enabled? If so, we'd like to have it.

File: scripts/devenv-builder/configure-vm.sh [hunk 3]
```diff
@@ -101,25 +101,26 @@ fi
...
-    curl -L -o "go${GO_VER}.linux-${GO_ARCH}.tar.gz" "https://go.dev/dl/go${GO_VER}.linux-${GO_ARCH}.tar.gz"
-    sudo rm -rf "/usr/local/go${GO_VER}"
-    sudo mkdir -p "/usr/local/go${GO_VER}"
-    sudo tar -C "/usr/local/go${GO_VER}" -xzf "go${GO_VER}.linux-${GO_ARCH}.tar.gz" --strip-components 1
-    sudo rm -rfv /usr/local/bin/{go,gofmt}
-    sudo ln --symbolic /usr/local/go${GO_VER}/bin/{go,gofmt} /usr/local/bin/
-    rm -rfv "go${GO_VER}.linux-${GO_ARCH}.tar.gz"
-fi
+
+# update to golang from centos-9-stream
```
Comment (ggiguash) [inline scripts/devenv-builder/configure-vm.sh:112]:
```suggestion
# Update to golang from centos-9-stream to make sure FIPS-compliant tools are used
```

File: scripts/devenv-builder/configure-vm.sh [hunk 4]
```diff
@@ -101,25 +101,26 @@ fi
 if ${INSTALL_BUILD_DEPS} || ${BUILD_AND_RUN}; then
     dnf_retry clean all
     dnf_retry update
-    dnf_retry install "gcc git golang cockpit make jq selinux-policy-devel rpm-build jq bash-completion avahi-tools"
+
+
+
```
Comment (ggiguash) [inline scripts/devenv-builder/configure-vm.sh:106]:
```suggestion
```

**Comment by dhellmann** [comment]
/hold

Let's talk about this approach before going ahead.

File: test/bin/scenario.sh [hunk 5]
```diff
@@ -488,6 +492,12 @@ load_global_settings() {
     fi
 
     REDHAT_AUTHORIZED_KEYS="$(cat "${SSH_PUBLIC_KEY}")"
+    cat - <<EOF | tee -a "${HOME}/.ssh/config"
+Host 192.168.*
```
Comment (ggiguash) [inline test/bin/scenario.sh:496]: This needs to be dynamically computed. Call get_vm_bridge_interface function and take X.Y. from the returned IP.

**Pull request closed without merging**